packages/host/app/components/operator-mode.gts

```diff
@@ -227,6 +227,9 @@ export default class OperatorMode extends Component<Signature> {
       cardComponentModifier: LinksToCardComponentModifier,
       optional: {
         stack: this.stack, // Not used currently, but eventually there will be more than one stack and we will need to know which one we are in.
+        addToStack: (card: Card) => {
```

Comment:
Because this action is specific to operator mode I have doubts `addToStack` belongs to `Actions`, so I placed it into the optional section of the context.